scripts/process_data.py

```diff
@@ -662,7 +207,17 @@ def main(self) -> None:
 
 
 @dataclass
-class ProcessMetashape:
+class _NoDefaultProcessMetashape:
+    """
+    Private class to order the parameters of ProcessMetashape in the right order for default values
+    """
+
+    xml: Path
+    """Path to the Metashape xml file."""
+
+
+@dataclass
+class ProcessMetashape(BaseConverterToNerfstudioDataset, _NoDefaultProcessMetashape):
```

Comment:
I haven't used those classes and I have no refactored them so there is still redundant code between them. Furthermore, I have never used them and I'm afraid of moving them in the main pages because they have no unit tests associated.